server/index.js

```diff
@@ -1,15 +1,119 @@
 require('dotenv/config');
+// const errorMiddleware = require('./error-middleware');
+// const staticMiddleware = require('./static-middleware');
```

Comment:
Please add these back in. You need error handling and static middleware for when we deploy your app.